lib/config/options/index.ts

```diff
@@ -1667,6 +1667,14 @@ const options: RenovateOptions[] = [
     default: false,
     supportedPlatforms: ['gitlab'],
   },
+  {
+    name: 'confidential',
+    description:
+      'If enabled, the issues and MRs created by Renovate are set as confidential. This is applicable to GitLab only',
```

Comment:
```suggestion
      'If enabled, issues created by Renovate are set as confidential.',
```

No need to list MRs/PRs if it's unsupported by all platforms